CHANGELOG.rst

```diff
@@ -16,8 +16,6 @@ Changelog
 * **BACKWARDS INCOMPATIBLE:** Invalid ASN.1 found during :doc:`/x509/index`
   parsing will raise an error on initial parse rather than when the invalid
   field is accessed.
-* **BACKWARDS INCOMPATIBLE:** Values passed to the :doc:`/x509/index` PEM
```

Comment:
Just to be clear, does the parser accept _leading_ extraneous data?

Reply:
Yes